vector/src/main/java/im/vector/app/features/matrixto/MatrixToBottomSheetViewModel.kt

```diff
@@ -296,8 +297,13 @@ class MatrixToBottomSheetViewModel @AssistedInject constructor(
         }
         viewModelScope.launch {
             try {
-                session.joinRoom(action.roomId, null, action.viaServers?.take(3) ?: emptyList())
-                _viewEvents.post(MatrixToViewEvents.NavigateToRoom(action.roomId))
+                session.joinRoom(action.roomIdOrAlias, null, action.viaServers?.take(3) ?: emptyList())
```

Comment:
to help give extra context it would be handy if the `null / true` arguments used named parameters  (or we could extract constants)